### ISSUE TITLE
Use WebP instead of PNG for spritesheets

### ISSUE THREAD

**Issue by Teoxoy** [Teoxoy/factorio-blueprint-editor#11]
Firefox and Safari don't support it yet
https://caniuse.com/#feat=webp

**Comment by baryluk** [comment]
I don't think it is worth it for sprites (it would not really save). It might be worth using for export of whole blueprint to WebP lossless, as it can save some space, but really not worth it either, because it is generated on the client anyway.

Just my 2 cents.

**Comment by Teoxoy** [comment]
Yeah, it might not be worth it. It was a consideration because the entity spritesheet is quite large (8k texture) but I've been experimenting with ways to load entity sprites on demand instead of a huge static spritesheet.
